api/v1/policyset_types.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package v1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PolicySetSpec defines the desired state of PolicySet
+type PolicySetSpec struct {
+	Description string `json:"description,omitempty"`
+	// +kubebuilder:validation:Required
+	Policies []string `json:"policies"`
```

Comment:
It'd also be nice to validate that the strings in the slice are not empty as well. You can do the custom string type that has `minLength` set on it or use Kustomize to modify after the fact like I did here:
https://github.com/stolostron/cert-policy-controller/commit/88127bde0b3f9a0164b9f6bf271a973610cab4f1#diff-b718b5e73b0beec0b5bf00caffbe16f247bcf005e8c2b8d827af5ece8f46962e